controllers/results/results_suite_test.go

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
wondering if this file is required although using it is pointed as "Best practices". Thoughts?

Reply:
file removed.